### PR TITLE
feat(vpc): vpc onboarding guide beta

### PR DESCRIPTION
**Pull request by RoRoJ**
### Description
Guide for VPC regional Private Networks beta testers (hidden from menu)

### CODE REVIEW

File: network/vpc/api-cli/vpc-private-beta.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You are part of the VPC Regional Private Networks Private Beta
+  - You have [created an API key](/console/my-project/how-to/generate-api-key/)
+</Message>
+
+## Getting started
+
+1. [Create a new Project](/console/my-project/how-to/create-a-project/). You may use an existing Project, but this is at your own risk. We recommend creating a new Project for the purposes of testing this beta product.
+
+2. Email the Project ID to vramiro@scaleway.com. You can find the Project ID in your Scaleway console at the URL `https://console.scaleway.com/<project-name>/settings`.
```
Comment (bene2k1) [inline network/vpc/api-cli/vpc-private-beta.mdx:27]: Maybe make put the email address into an image to avoid spam bots scrapping it too easily?

Comment (RoRoJ) [reply network/vpc/api-cli/vpc-private-beta.mdx]: Done!

File: network/vpc/api-cli/vpc-private-beta.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  - You are part of the VPC Regional Private Networks Private Beta
+  - You have [created an API key](/console/my-project/how-to/generate-api-key/)
+</Message>
+
+## Getting started
+
+1. [Create a new Project](/console/my-project/how-to/create-a-project/). You may use an existing Project, but this is at your own risk. We recommend creating a new Project for the purposes of testing this beta product.
+
+2. Email the Project ID to <Lightbox src="scaleway-email-add.webp" size="large" />
+
```
Comment (vramiro) [inline network/vpc/api-cli/vpc-private-beta.mdx:28]: This whole section 2 can be removed now that we have the migration procedure.
People can either create a project and add regional PNs using the API or else migrate their own existing resources

**Review by vramiro — CHANGES_REQUESTED** [review]

File: network/vpc/api-cli/vpc-private-beta.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  posted: 2021-02-14
+categories:
+  - network
+---
+
+VPC Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You are part of the [VPC Regional Private Networks Private Beta](https://www.scaleway.com/en/betas/#regional-private-network)
```
Comment (nedaehm) [inline network/vpc/api-cli/vpc-private-beta.mdx:19]: should the link be "/betas/#regional-private-network"?

**Review by vramiro — APPROVED** [review]
LGTM

**Pull request merged**